ui/raidboss/data/06-ew/raid/p2s.ts

```diff
@@ -0,0 +1,361 @@
+import Conditions from '../../../../../resources/conditions';
+import NetRegexes from '../../../../../resources/netregexes';
+import Outputs from '../../../../../resources/outputs';
+import { Responses } from '../../../../../resources/responses';
+import ZoneId from '../../../../../resources/zone_id';
+import { RaidbossData } from '../../../../../types/data';
+import { NetMatches } from '../../../../../types/net_matches';
+import { TriggerSet } from '../../../../../types/trigger';
+
+// TODO: Callout cardinal for Spoken Cataract
+// TODO: Debuff collect for Marks and callouts for those without debuff
+// TODO: Add cardinal to Channeling Flow
+// TODO: Fix headmarker ids for Kampeos Harma Callouts
+
+export interface Data extends RaidbossData {
+  flareTarget?: string;
+  decOffset?: number;
+}
+
+// Due to changes introduced in patch 5.2, overhead markers now have a random offset
+// added to their ID. This offset currently appears to be set per instance, so
+// we can determine what it is from the first overhead marker we see.
+// The first 1B marker in the encounter is an Ominous Bubbling (00A1).
+const firstHeadmarker = parseInt('00A1', 16);
+const getHeadmarkerId = (data: Data, matches: NetMatches['HeadMarker']) => {
+  // If we naively just check !data.decOffset and leave it, it breaks if the first marker is 00DA.

```

Comment:
I think this might be why I didn't see Limit Cut callouts? The first head marker present in the encounter is not Ominous Bubbling, it's Doubled Impact, `0103`.

Reply:
I've updated it to use Doubled Impact. When I originally wrote that trigger, it used IDs that come from the random offset. It did not call the numbers correct there.